src/q_and_a_service/queries.js

```diff
@@ -0,0 +1,5 @@
+const getQA = "SELECT * FROM product_question";
```

Comment:
It might be a good idea to wrap this in an async function or promise with a dedicated `.catch()` statement to handle any errors that might occur when accessing the database